### PR TITLE
Add "Go to Centova Cast" button

### LINKED ISSUE #48: Go to Centova button
in the line to replace WHMCS maybe a go to Centova button in the menu can be handy, maybe password reset too?

### PR DESCRIPTION
**Pull request by meyskens**
schermafbeelding 2016-12-31 om 16 36 12

*Needs an ITFrame update to work*

Fixes #48 

---
This change is [Reviewable](https://reviewable.io/reviews/innovate-technologies/control/68)

### CODE REVIEW

**Comment by meyskens** [comment]
Tested and working

File: src/manage/manage.html [hunk 1]
```diff
@@ -20,6 +20,16 @@
         </li>
       </ul>
     </div>
+    <ul class="panel-body nav nav-pills nav-stacked" ng-show="service.group.toLowerCase().indexOf('node') === -1">
```
Comment (leoetlino) [inline src/manage/manage.html]: Can you hide this if !service.username or !service.password?

Comment (meyskens) [inline src/manage/manage.html]: sure!

File: src/manage/manage.html [hunk 2]
```diff
@@ -20,6 +20,16 @@
...
+      <li>
+        <form method="post" action="{{ 'https://' + service.server + '.shoutca.st/login/index.php' | lowercase }}" id="go-to-centova">
+        <input type="hidden" name="username" ng-value="service.username" />
```
Comment (leoetlino) [inline src/manage/manage.html]: You don't need the self-closing part, e.g. this can just be `<input ...>`

Comment (meyskens) [inline src/manage/manage.html]: But XHTML was nice. Will change

**Comment by leoetlino** [comment]
LGTM

**Pull request merged**